package.json

```diff
@@ -50,8 +50,8 @@
     "cheerio": "1.0.0-rc.2",
     "xmldom": "0.1.27",
     "logops": "2.1.0",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master"
+    "iotagent-node-lib": "git://github.com/dcalvoalonso/iotagent-node-lib.git#task/fixWarnings",
```

Comment:
Must be restored once https://github.com/telefonicaid/iotagent-node-lib/pull/784 is merged.

Reply:
It has been merded. Could you change, pls?